lib/classes/PluginManager.js

```diff
@@ -309,6 +310,26 @@ class PluginManager {
     });
   }
 
+  loadVariableResolvers(pluginInstance) {
+    for (const [variablePrefix, resolverOrOptions] of _.entries(
+      pluginInstance.variableResolvers || {}
+    )) {
+      let options = resolverOrOptions;
+      if (typeof resolverOrOptions === 'function') {
+        options = {
+          resolver: resolverOrOptions,
+        };
+      }
+      this.serverless.variables.variableResolvers.push({
+        regex: new RegExp(`^${variablePrefix}:`),
+        method: options.resolver.bind(this.serverless.variables),
+        custom: true,
```

Comment:
Maybe `isCustom` (to indicate it's a boolean, and share same convention as `isDisabledAtPrepopulation`)